crates/sovereign-sdk/module-system/sov-state/src/zk_storage.rs

```diff
@@ -149,11 +145,11 @@ where
     }
 
     fn open_proof(
-        state_root: Self::Root,
-        state_proof: StorageProof<Self::Proof>,
+        state_root: StorageRootHash,
+        state_proof: StorageProof,
     ) -> Result<(StorageKey, Option<StorageValue>), anyhow::Error> {
         let StorageProof { key, value, proof } = state_proof;
-        let key_hash = KeyHash::with::<H>(key.as_ref());
+        let key_hash = KeyHash::with::<DefaultHasher>(key.as_ref());
```

Comment:
was it the case that we were passing DefaultHasher for previous zk storage? I expect it to be that way but let's confirm

Reply:
We passed DefaultHasher everywhere but the type of it was same for mock/not mock stuff. So there is no use in this generic at all.